x-pack/plugins/apm/server/feature.ts

```diff
@@ -60,3 +60,9 @@ export const APM_FEATURE = {
 
 export const APM_SERVICE_MAPS_FEATURE_NAME = 'APM service maps';
 export const APM_SERVICE_MAPS_LICENSE_TYPE = 'platinum';
```

Comment:
Since a feature name and a feature license belongs together I think we should group them like:

```suggestion
export const APM_SERVICE_MAPS_FEATURE = {
 name: 'APM service maps',
 license: 'platinum'
}
```